pytest_django/plugin.py

```diff
@@ -399,12 +399,54 @@ def teardown():
         request.addfinalizer(teardown)
 
 
-@pytest.fixture(autouse=True, scope='function')
-def _django_clear_outbox(django_test_environment):
-    """Clear the django outbox, internal to pytest-django."""
-    if django_settings_is_configured():
-        from django.core import mail
-        del mail.outbox[:]
+class _DirectMailboxAccessProtector(list):
+
+    def _raise_assertion(*args, **kwargs):
+        raise AssertionError('To access mail.outbox, use the mailoutbox fixture.')
+
+    __len__ = __getitem__ = __nonzero__ = __bool__ = _raise_assertion
+
+
+# def mailoutbox_setup_and_teardown(outbox_type):
+#
+
+
+@pytest.yield_fixture(autouse=True)
```

Comment:
Shouldn't this be `scope="session"`? If it's session-scoped then the "protection" would be on for other fixtures too.